x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/display_settings/constants.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const LEAVE_UNASSIGNED_FIELD = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.contentSources.displaySettingsLogic.leaveUnassignedField',
```

Comment:
Small change here but we don't use component names in these; just a contextual path so instead of `displaySettingsLogic `, let's stick with `displaySettings`

Reply:
Gotcha, thanks! Updated in the following commit.
Please continue being vocal in comments, I feel almost blind in the codebase so far. 🙈